mne/preprocessing/_beer_lambert_law.py

```diff
@@ -124,9 +124,22 @@ def _probe_distances(raw):
     return np.array(dist, float)
 
 
-def _short_channels(raw, threshold=0.01):
-    """Return a vector indicating which channels are short.
+def short_channels(raw, threshold=0.01):
```

Comment:
sorry for coming late to the battle field but I don't like the name `short_channels `
especially in the preprocessing namespace. 2 options:

- rename to short_nirs_channels
- move nirs preprocessing channels to their own submodule mne.preprocessing.nirs (if they are many to come)

Reply:
also it should take info as input so it can be called on epochs or evoked etc.